samples/highcharts/chartchooser/continuous-distribution-bell-curve-standard-deviation/demo.js

```diff
@@ -0,0 +1,146 @@
+var data = [
+    111,
+    107,
+    100,
+    107,
+    114,
+    115,
+    111,
+    97,
+    100,
+    112,
+    104,
+    89,
+    104,
+    102,
+    91,
+    114,
+    114,
+    103,
+    106,
+    105,
+    113,
+    109,
+    108,
+    113,
+    130,
+    128,
+    128,
+    118,
+    113,
+    120,
+    132,
+    111,
+    124,
+    127,
+    128,
+    136,
+    106,
+    118,
+    119,
+    123,
+    124,
+    126,
+    116,
+    127,
+    119,
+    97,
+    86,
+    102,
+    110,
+    120,
+    103,
+    115,
+    93,
+    72,
+    111,
+    103,
+    123,
+    79,
+    119,
+    110,
+    110,
+    107,
+    74,
+    105,
+    112,
+    105,
+    110,
+    107,
+    103,
+    77,
+    98,
+    90,
+    96,
+    112,
+    112,
+    114,
+    93,
+    106
+];
+
+var pointsInInterval = 5;
+
+Highcharts.chart("container", {
+    chart: {
+        margin: [50, 0, 50, 50],
+        events: {
+            load: function () {
+                Highcharts.each(this.series[0].data, function (point, i) {
```

Comment:
`Highcharts.each` is deprecated. Use `this.series[0].data.forEach`.

Reply:
Done 👍